rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -173,7 +178,8 @@ class rocalTensorInfo {
     RocalROIType roi_type() const { return _roi_type; }
     RocalTensorDataType data_type() const { return _data_type; }
     RocalTensorlayout layout() const { return _layout; }
-    RocalROI *get_roi() const { return (RocalROI *)_roi_buf; }
+    RocalROI *get_roi() const { return (RocalROI *)_roi.get(); }
+    std::shared_ptr<std::vector<float>> get_sample_rate() const { return _sample_rate; }
```

Comment:
Where this get_sample_rate used in audio pipeline?

Reply:
In the resample node